web/context.go

```diff
@@ -26,19 +26,19 @@ type Context struct {
 
 // LogAuditRec logs an audit record using default RestLevel.
 func (c *Context) LogAuditRec(rec *audit.Record) {
-	c.LogAuditRecWithLevel(rec, app.RestLevel)
+	c.LogAuditRecWithLevel(rec, app.LevelAPI)
 }
 
 // LogAuditRec logs an audit record using specified Level.
```

Comment:
```suggestion
// LogAuditRecWithLevel logs an audit record using specified Level.
```

Should we also mention that the level gets overridden in case of a permission error?

Reply:
Fixed via https://github.com/mattermost/mattermost-server/pull/15076/commits/04e503a93b678005c9c71ebd5ad897bede654ce1